torchci/lib/bot/pytorchBotHandler.ts

```diff
@@ -283,9 +285,7 @@ The explanation needs to be clear on why this is needed. Here are some good exam
 
     if (
       rebase &&
-      !(await this.hasWorkflowRunningPermissions(
-        this.ctx.payload?.comment?.user?.login
-      ))
+      !(await this.hasRebasePermissions(this.ctx.payload?.comment?.user?.login))
```

Comment:
preserves the old behavior, but I think if they already had ciflow/trunk it would not need approval to run trunk on the rebased version, which is probably bad

Reply:
I think we need to redefine the strategy here when it comes to rebase given that CI approval is now enforced by default.  We should allow people to rebase all the times.

1. If they already has write permission, of course rebasing is accepted
1. If they are contributors without write permission or are 1st-time contributors, rebasing their PRs should be ok because, like submitting a new commit, CI approval is needed again anyway.  A potential rough edge here is that any `ciflow/*` in the PR are removed but allowing people to do rebase always is still an improvement nonetheless

I will add this as a discussion topic in the upcoming design discussion.  For now, we could keep the current behavior.